community/fzy-leo-arch/version

```diff
@@ -0,0 +1 @@
+git 1
```

Comment:
Peg this to the latest commit as the author might push more experimental changed

Reply:
done 0e7a0db6